validate_sptx/validate_sptx.py

```diff
@@ -54,44 +54,38 @@ def validate(experiment_json: str, fuzz: bool=False) -> bool:
         valid &= experiment_validator.validate_object(experiment, name)
 
     # validate manifests that it links to.
-    possible_manifests = []
+    manifests = []
     manifest_validator = SpaceTxValidator(_get_absolute_schema_path('fov_manifest.json'))
     with backend.read_contextmanager(experiment['primary_images']) as fh:
-        possible_manifests.append((json.load(fh), experiment['primary_images']))
+        manifests.append((json.load(fh), experiment['primary_images']))
 
     # loop over all the manifests that are stored in auxiliary images. Disallowed names will
     # have already been excluded by experiment validation.
     for manifest in experiment['auxiliary_images'].values():
         with backend.read_contextmanager(manifest) as fh:
-            possible_manifests.append((json.load(fh), manifest))
+            manifests.append((json.load(fh), manifest))
 
-    # we allow the objects linked from primary_images and auxiliary images to either be
-    # manifests OR field_of_view files. We distinguish these by checking if they have a `contents`
-    # flag, which indicates it is a manifest.
     fovs = []
-    for manifest, filename in possible_manifests:
-        if 'contents' in manifest:  # is a manifest; validate
-            if fuzz:
-                manifest_validator.fuzz_object(manifest, filename)
-            else:
-                valid &= manifest_validator.validate_object(manifest, filename)
-
-            # contains fields of view
-            for key, fov in manifest['contents'].items():
-                with backend.read_contextmanager(fov) as fh:
-                    fovs.append((json.load(fh), fov))
-
-        else:  # manifest is a field of view
-            fovs.append((manifest, None))
-
-    # validate fovs
-    assert len(fovs) != 0
-    fov_validator = SpaceTxValidator(_get_absolute_schema_path('field_of_view/field_of_view.json'))
-    for fov, filename in fovs:
+    for manifest, filename in manifests:
```

Comment:
shouldn't this validate against fov_manifest.json?

Reply:
Hmmm.... I might be missing something, but that's what I thought it was doing:

```
    manifest_validator = SpaceTxValidator(_get_absolute_schema_path('fov_manifest.json'))
...
    for manifest, filename in manifests:
...
            if not manifest_validator.validate_object(manifest, filename):
                valid = False
```